docs/api/qiskit/0.45/qiskit.algorithms.eigensolvers.VQD.mdx

```diff
@@ -91,16 +85,6 @@ python_api_name: qiskit.algorithms.eigensolvers.VQD
     [list](https://docs.python.org/3/library/stdtypes.html#list "(in Python v3.12)")\[[float](https://docs.python.org/3/library/functions.html#float "(in Python v3.12)")]
   </Attribute>
 
-  ###
-
-  <Attribute name="initial point">
-    An optional initial point (i.e. initial parameter values) or a list of initial points (one for every k-th eigenvalue) for the optimizer. If `None` then VQD will look to the ansatz for a preferred point and if not will simply compute a random one.
-
-    **Type**
-
-    Sequence\[[float](https://docs.python.org/3/library/functions.html#float "(in Python v3.12)")] | Sequence\[Sequence\[[float](https://docs.python.org/3/library/functions.html#float "(in Python v3.12)")]] | None
-  </Attribute>
```

Comment:
Should this have been removed completely? Just going off the class description but it seems like it should still be listed

Reply:
Yes, this is a consequence of this [PR](https://github.com/Qiskit/qiskit/pull/12162). The attribute in the original file was missing a hyphen (`initial point` instead of `initial_point`). If we add the hyphen, sphinx complained that the attribute is duplicated and we decided to remove it entirely [here](https://github.com/Qiskit/qiskit/pull/12162#discussion_r1557834962)